README.md

```diff
@@ -148,7 +148,9 @@ end
 
 # Standard card
 intent "SendStandardCard" do
-  standard_card = card("Hello World", "I'm alive!", "https://placehold.it/200")
+  image = { small: "http://placehold.it/720x480", large: "http://placehold.it/1200x800" }
+  standard_card = card("Hello World", "I'm alive!", image)
```

Comment:
Thoughts on using two parameters for image urls (both optional), which might avoid the breaking change?

Reply:
I did consider this, but figured that would, in itself still be a breaking change - modifying the functionality within the card of providing the same url for both small and large images?

Reply:
So it would be

```ruby
standard_card = card("Hello World", "I'm alive!", "http://placehold.it/720x480", "http://placehold.it/1200x800")
```

And `card` would make the final parameter default to the small image if not present?

I'm generally keen to keep the most-used interfaces as 'stringy' as possible. I'm happy with directives to be an array but if we can avoid users having to manually construct hashes, that feels good to me.

Reply:
Updated as per your request 👍